spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -296,15 +296,18 @@ An event parameter may have multiple qualifiers.
 public void afterDocumentUpdatedByAdmin(@Observes @Updated @ByAdmin Document doc) { ... }
 ----
 
-Then this observer method will only be notified if all the observed event qualifiers are specified when the event is fired:
+Then this observer method will be notified if the set of observer qualifiers is a subset of the fired event's qualifiers:
 
 [source, java]
 ----
 @Inject Event<Document> documentEvent;
 ...
-documentEvent.select(new UpdatedQualifier(), new ByAdminQualifier()).fire(document);
+documentEvent.select(new UpdatedQualifier(), new ByAdminQualifier(), new ClarificationQualifier()).fire(document);
 ----
 
+In the above example the event is fired with `@ByAdmin`, `@Updated` and `@Clarification` qualifiers while the observer qualifiers are `@Updated` and `@ByAdmin`.
+Observer qualifiers therefore form a subset of event qualifiers and the observer will be notified.
```

Comment:
@antoinesd well ... empty set is a subset of all other sets and itself . The question is, whether Average Joe knows this :)
Anyway, the example just below this text shows that
`public void afterDocumentEvent(@Observes Document doc) { ... }` will also be notified, which is exactly the case you are reffering to.